deta/_async/client.py

```diff
@@ -0,0 +1,167 @@
+import typing
+
+import os
+import aiohttp
+from urllib.parse import quote
+
+from deta.utils import _get_project_key_id
+from deta.base import FetchResponse, Util
+
+
+class AsyncDeta:
```

Comment:
I don't think we should have an `AsyncDeta` class as it is redundant and does not add anything new/different to the `Deta` class. We can just do: 
```python
from deta import Deta

async_base = Deta("project_key").AsyncBase("base_name")
```

And
```python
from deta import AsyncBase

async_base = AsyncBase("base_name")
```

Reply:
Make sense. I'll change it.

Reply:
Removed.

Reply:
Updated. Check it, please.